test/srt/test_double_sparsity.py

```diff
@@ -0,0 +1,95 @@
+import os
+import unittest
+from types import SimpleNamespace
+
+import requests
+
+from sglang.srt.utils import kill_child_process
+from sglang.test.run_eval import run_eval
+from sglang.test.test_utils import (
+    DEFAULT_MODEL_NAME_FOR_TEST,
+    DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+    DEFAULT_URL_FOR_TEST,
+    popen_launch_server,
+)
+
```

Comment:
add this to https://github.com/sgl-project/sglang/blob/869f1c02c4a7140c674ea92127a45eac0211bf74/test/srt/run_suite.py#L13